prov/verbs/src/verbs_domain.c

```diff
@@ -269,6 +269,9 @@ static int fi_ibv_domain_close(fid_t fid)
 		rdma_destroy_ep(domain->rdm_cm->listener);
 		rdma_destroy_event_channel(domain->rdm_cm->ec);
 		free(domain->rdm_cm);
+
+		if (domain->eq)
```

Comment:
domain->eq is set before we know if async_add() succeeds, so this doesn't indicate that add worked.